packages/platforms/inventec/x86-64/x86-64-inventec-d7032q28b/modules/builds/gpio-ich.c

```diff
@@ -0,0 +1,548 @@
+/*
```

Comment:
Please don't include copies of existing kernel modules in your platform. 
You can enable CONFIG_GPIO_ICH=m in the 3.16 kernel config if its not currently available.

Reply:
Thanks for reviewing.  Got it.  I have a few questions.
Changing the kernel config will affect all platforms? Is this what we want?
If I want to modify kernel source, I will need to create a patch, right? Does that means my patch will be applied for all platforms too? Can we select the patches to apply for a particular platform?
If our platform kernel module under packages/platforms/<vendor>/x86-64/x86-64-<vendor>-<platform>/modules/builds/  consists of multiple .c files? How do I modify the Makefile? I don't see any example of this.  It seems to me each kernel module can only have one .c in ONL.  Can I put their .h files in the same directory as .c files?  Thanks in advance.